frontend/src/components/TaskValueCreatedVisualization.module.scss

```diff
@@ -5,7 +5,9 @@
   @extend .layout-row;
   gap: 47px;
   width: 100%;
+  height: 100%;
   .stakesDescription {
     width: 100%;
+    height: 100%;
```

Comment:
Is there a reason to define width and height twice here with the same values?

Reply:
Seems like the whole .stakesDescription class is unnecessary